emptytheme/style.css

```diff
@@ -0,0 +1,15 @@
+/*
+Theme Name: Empty Theme
```

Comment:
Should we allow all this header information to be specified in theme.json? @aristath, @mtias, @nosolosw

It seems strange that we force a theme to have a CSS file. I guess a theme with just theme.json should be possible.

Reply:
I was thinking the same. That would be nice.

Reply:
The theme directory (and core as well) would need, what I assume, to be a pretty sizable overhaul to do away with the need to have any stylesheet.

Reply:
Yeah, currently core detects themes from their style.css file. So if we want to allow these to be inside the `theme.json` file, a patch would have to be submitted to core at the same time so that these can be properly used.
I agree though, the stylesheet will in many cases be just a just file used to define the theme.

Reply:
I guess as theme.json involves we should allow just a theme.json theme. But this change would touch many parts of our code and is not something essential. Also as we are I guess most themes will still need at least some lines of CSS code. But once theme.json expands having theme information there seems like a good path.